chart/values.schema.json

```
@@ -613,6 +613,12 @@
                 }
             ]
         },
+        "useDefaultImageForMigration": {
+            "description": "To avoid images with user code for running and waiting for DB migrations set this to ``true``. ",
+            "type": "boolean",
+            "x-docsSection": "Airflow",
```

```suggestion
            "x-docsSection": "Images",
```

I feel like this is a better place?